code/core/write_sdl_ntuple.cc

```diff
@@ -703,7 +703,7 @@ void setGnnNtupleBranches(SDL::Event* event)
         ana.tx->pushbackToBranch<vector<int>>("tc_lsIdx", lsIdx);
     }
 
-    std::cout <<  " ana.tx->getBranch<vector<vector<int>>>('tc_lsIdx').size(): " << ana.tx->getBranch<vector<vector<int>>>("tc_lsIdx").size() <<  std::endl;
+    // std::cout <<  " ana.tx->getBranch<vector<vector<int>>>('tc_lsIdx').size(): " << ana.tx->getBranch<vector<vector<int>>>("tc_lsIdx").size() <<  std::endl;
```

Comment:
I would propose to delete this. No reason to leave comments in the code (especially if they crash the code when enabled).